runtime/Go/antlr/atn_config_set.go

```diff
@@ -269,48 +284,59 @@ func (b *BaseATNConfigSet) ContainsFast(item ATNConfig) bool {
 	return b.configLookup.contains(item) // TODO: containsFast is not implemented for Set
 }
 
+// Clear removes all contained objects from this set.
 func (b *BaseATNConfigSet) Clear() {
 	if b.readOnly {
 		panic("set is read-only")
 	}
 
 	b.configs = make([]ATNConfig, 0)
 	b.cachedHash = -1
-	b.configLookup = NewSet(nil, equalATNConfigs)
+	b.configLookup = newSet(nil, equalATNConfigs)
 }
 
+// FullContext returns true if this set is a full context
 func (b *BaseATNConfigSet) FullContext() bool {
 	return b.fullCtx
 }
 
+// GetDipsIntoOuterContext returns true is this set dips into it's outer
+// context.
 func (b *BaseATNConfigSet) GetDipsIntoOuterContext() bool {
 	return b.dipsIntoOuterContext
 }
 
+// SetDipsIntoOuterContext sets whether this set dips into it's outer context.
 func (b *BaseATNConfigSet) SetDipsIntoOuterContext(v bool) {
 	b.dipsIntoOuterContext = v
 }
 
+// GetUniqueAlt returns this set's unique alternative.
 func (b *BaseATNConfigSet) GetUniqueAlt() int {
 	return b.uniqueAlt
 }
 
+// SetUniqueAlt sets the unique alternative for this set.
```

Comment:
It's unclear from this what a unique alternative is, and how an int represents that. Same for Get* above.